frontend/src/component/changeRequest/ChangeRequest/StrategyTooltipLink/StrategyTooltipLink.tsx

```diff
@@ -77,34 +107,18 @@ export const StrategyTooltipLink: FC<IStrategyTooltipLinkProps> = ({
     <StyledContainer>
         <GetFeatureStrategyIcon strategyName={change.payload.name} />
         <Truncated>
-            <ConditionallyRender
-                condition={Boolean(
-                    (previousTitle && previousTitle !== change.payload.title) ||
-                        (!previousTitle && change.payload.title)
-                )}
-                show={
-                    <Truncated>
-                        <Typography component="s" color="text.secondary">
-                            {previousTitle ||
-                                formatStrategyName(change.payload.name)}
-                        </Typography>{' '}
-                    </Truncated>
-                }
-            />
-            <Truncated>
-                <TooltipLink
-                    tooltip={children}
-                    tooltipProps={{
-                        maxWidth: 500,
-                        maxHeight: 600,
-                    }}
-                >
-                    <Typography component="span">
-                        {change.payload.title ||
-                            formatStrategyName(change.payload.name)}
-                    </Typography>
-                </TooltipLink>
-            </Truncated>
+            <TooltipLink
+                tooltip={children}
+                tooltipProps={{
+                    maxWidth: 500,
+                    maxHeight: 600,
+                }}
+            >
+                <Typography component="span">
+                    {formatStrategyName(change.payload.name)}
+                </Typography>
+            </TooltipLink>
+            {<StrategyName change={change} previousTitle={previousTitle} />}
```

Comment:
I'm not sure if we should couple strategy name to the entire change request. Maybe the API should be: strategyName, oldTitle, newTitle to express what really matters